### DISCUSSION TITLE
X account problem

### DISCUSSION THREAD

**Discussion by Ugurkayaaa** [IBAX-io/go-ibax#2146] [Q&A] [answered]
Hello, my X account has been suspended. It has been suspended for about 4 months. I can't log in to my account right now. I have my wallet information, but I can't log in because I can't open account x. I would appreciate your help.

**Comment by yolandadadada** [comment, marked as answer]
You can try and write a daily email to twitter official to request unblocking, we had a user unblock his twitter this way before.
